src/Fabulous/Program.fs

```diff
@@ -19,10 +26,11 @@ type Program<'arg, 'model, 'msg> =
         ExceptionHandler: exn -> bool
     }
 
+//TODO how is this different to the above? What's a 'marker? what's the 'arg?
```

Comment:
`'marker` here is the type of the view. This is used by Fabulous.MauiControls and Fabulous.Avalonia to strongly-type the view being returned.

For example, a `ListView` only accepts "cells" (anything inheriting from `Cell` type like `TextCell`, `ImageCell` and so on). You don't want a page in a ListView. In Fabulous, this is represented by the marker interfaces: `IFabTextCell`, `IFabImageCell` which both inherit from `IFabCell`. `IFabPage` doesn't inherit from `IFabCell` making it impossible to be used inside a ListView.

Reply:
`'marker` in Program type is mainly used by components to signal that this component will return that specific view type and can only be used in a specific way.

We also use this `'marker` type when running an application. `UseFabulousApp` will require `'marker` to be `IFabApplication` because Maui/Avalonia requires an Application type and not some random Label for example.